src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -1277,13 +1277,19 @@ namespace OpenMS
     iso_similarity_ = param_.getValue("isotopic_similarity").toBool();
 
     // use defaults if empty for all .tsv files
-    db_mapping_file_ = ListUtils::toStringList<std::string>(param_.getValue("db:mapping"));
-    if (db_mapping_file_.empty()) db_mapping_file_ = ListUtils::toStringList<std::string>(defaults_.getValue("db:mapping"));
-    db_struct_file_ = ListUtils::toStringList<std::string>(param_.getValue("db:struct"));
-    if (db_struct_file_.empty()) db_struct_file_ = ListUtils::toStringList<std::string>(defaults_.getValue("db:struct"));
+    db_mapping_file_ = param_.getValue("db:mapping").toStringVector();
+    if (db_mapping_file_.empty()) db_mapping_file_ = param_.getValue("db:mapping").toStringVector();
+    db_struct_file_ = param_.getValue("db:struct").toStringVector();
+    if (db_struct_file_.empty()) db_struct_file_ = param_.getValue("db:struct").toStringVector();
 
+    /*
     pos_adducts_fname_ = param_.getValue("positive_adducts").toString();
     neg_adducts_fname_ = param_.getValue("negative_adducts").toString();
+    */
+
+    //TODO with empty adducts, do we fall back to defaults, or do we really assume no adducts should be searched?
```

Comment:
I do not know any use case "no adducts" would be used, since usually most of the compounds are protonated. Nonetheless, I would probably allow it, but throw a warning, that no adducts have been specified.

Reply:
Agreed.
One can imagine that the spectrum was preprocessed to only contain actual compound masses (no adduct, charge 1) and then searching without adducts would be meaningful.

Reply:
yep... e.g., if these are unknowns